pages.fr/common/topydo.md

```diff
@@ -0,0 +1,28 @@
+# topydo
+
+> Une application de liste de choses à faire qui utilise le format todo.txt.
+> Plus d'informations: <https://github.com/topydo/topydo>.
+
+- Ajouter une tâche à un projet spécifique avec un contexte donné:
```

Comment:
I think this is the correct form of the présent de l'indicatif
```suggestion
- Ajoute une tâche à un projet spécifique avec un contexte donné:
```

Reply:
`ajouter` (infinitive) is OK, it follows the [style guide's "imperative mood"](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood).